README.md

```diff
@@ -150,7 +150,7 @@ Each day I take one subject from the list below, read it cover to cover, take no
 - [x] [The Machine Learning Mastery Method](http://machinelearningmastery.com/machine-learning-mastery-method/)
 - [x] [Machine Learning for Programmers](http://machinelearningmastery.com/machine-learning-for-programmers/)
 - [x] [Applied Machine Learning with Machine Learning Mastery](http://machinelearningmastery.com/start-here/)
-- [ ] [Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
+- [ ] [(In Progress) Python Machine Learning Mini-Course](http://machinelearningmastery.com/python-machine-learning-mini-course/)
```

Comment:
matplotlib.pyplot - followed the tutorial here:
http://matplotlib.org/users/pyplot_tutorial.html

import matplotlib.pyplot as plt
plt.plot([1,2],[1,2]) -- will create a line with points (1,1), (2,2)
Can use matlab style line styling
plt.show() will display the plot.
plt.axes() to configure axes
plt.setp(lines) to conifgure lines

use figure() to create a grid of plots.

plt.text(60, .025, r'$\mu=100,\ \sigma=15$') - to display text on the plot at the given location (60, .025)
text() supports TeX expressions.

annotate() will annotate a value on the line.
plt.annotate('local max', xy=(2, 1), xytext=(3, 1.5),
            arrowprops=dict(facecolor='black', shrink=0.05),
            )

Scale can change to log and logit:
plt.xscale(‘log’)